packages/data-store/src/entities/xstate/StateEntity.ts

```diff
@@ -0,0 +1,47 @@
+import { BaseEntity, Column, CreateDateColumn, Entity, PrimaryGeneratedColumn, UpdateDateColumn } from 'typeorm'
+import { NonPersistedXStateStoreEvent } from '../../types'
+
+@Entity('StateEntity')
+export class StateEntity extends BaseEntity {
+  @PrimaryGeneratedColumn('uuid', { name: 'id' })
+  id!: string
+
+  @Column({ name: 'step', nullable: false })
+  step!: string
+
+  @Column({ name: 'type', nullable: false })
+  type!: string
+
+  @Column({ name: 'event_name', nullable: false })
```

Comment:
Elsewhere it is called an XStateEventType, which makes more sense to me

Reply:
changed to `xStateEventType` with column name: `xstate_event_type`